notebooks/MAST/downloading_transiting_planet_data/downloading_exoplanet_data.ipynb

```diff
@@ -0,0 +1,1287 @@
+{
```

Comment:
<p>I typically don't use this package (I only ugly print I guess, lol) so it would be good to explain what this code block does.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>